apps/testnorge-tp/nais.yaml

```diff
@@ -32,3 +32,18 @@ spec:
   image: {{ image }}
   ingresses:
     - "https://testnorge-tp.dev.intern.nav.no"
+  azure:
+    application:
+      enabled: true
+      tenant: nav.no
+  accessPolicy:
+    inbound:
+      rules:
+        - application: testnorge-statisk-data-forvalter
+        - application: testnorge-hodejegeren
```

Comment:
Hodejegeren går ikke mot testnorge-tp, den brukes kun av statisk-data-forvalter

Reply:
Du må også legge til i inbound rules (dette må også legges inn for testnorge-statisk-data-forvalter):
```
        - application: team-dolly-lokal-app
          cluster: dev-fss
        - application: testnav-oversikt-frontend
          cluster: dev-gcp
```

Reply:
> Hodejegeren går ikke mot testnorge-tp, den brukes kun av statisk-data-forvalter

testnorge-tp har HodejegerenConsumer som er brukt i TpService.getLivingIdentities()